misaligned.c

```diff
@@ -16,6 +16,9 @@ int printColorMap() {
 int main() {
     int result = printColorMap();
     assert(result == 25);
+    //assert(result == 0);
+    //assert(result == 1);
+    assert(result == 16);

```

Comment:
is result == 16 expected behavior from production code?
Also how can result be 25 and subsequently 16 without any production code execution?
Since we need to write the expectation first (without implementation) tests are bound to fail. Having asserts with incorrect values in order to fail the test is not right!